apps/root/src/pages/position-detail/components/summary-container/index.tsx

```diff
@@ -53,9 +155,25 @@ const PositionSummaryContainer = ({ position, pendingTransaction, isLoading }: P
             <GraphContainer position={position} isLoading={isLoading} />
           </Grid>
           <Grid item xs={12}>
-            <StyledPaper>
-              <PositionSwaps position={position} isLoading={isLoading} />
-            </StyledPaper>
+            <PositionTimelineControls
```

Comment:
could we abstract this in just one component? Since it needs to define a lot of things here